### PR TITLE
mac m1s use arm64 pinot container

### PR DESCRIPTION
**Pull request by jacksonargo**
Use the arm64 pinot container when running unit tests on an M1 laptop.

### CODE REVIEW

**Review by cyrilou242 — CHANGES_REQUESTED** [review]
LGTM
not easy to be the first developer running the project on a M1! 

- let's move reader.close() to a `finally` close. this ensures the reader is closed event if readLine fails
```
try {
...
} catch {
...
}finally {
reader.close()
}
```

- L74 let's not catch silently. you could log an error or throw. I'm in favor of throwing and letting the next developer fix the issue, it's unlikely we want to pursue with an incorrect image.  

few java conventions: 
method names start with lowercase:
`ImageTag()` `HostArch()` --> `imageTag()`, ...

in this codebase we try to put `final` when a variable is effectively final. 
this makes the code more robust to changes. 

`final` prevents [the reference hold by an object] / [the value of a primitive] to be changed

**Comment by jacksonargo** [comment]
* I renamed the methods w/ lowercase.
* Catch block throws a runtime exception. My original thought was to let it try the x86_64 image if the systctl command fails, but ya it _shouldn't_ fail, and if it does, it's prob best to surface it.
* Since the catch block is throwing a runtime exception, I left the reader.Close() as is.

File: pinot-test-container/src/main/java/org/apache/pinot/testcontainer/PinotContainer.java [hunk 1]
```diff
@@ -139,7 +172,10 @@ public void addTables() throws IOException, InterruptedException {
 
   private long getTableSize(final String tableName) throws IOException, InterruptedException {
     final ExecResult checkIngestionIsFinished = this.execInContainer(
-        "curl", "-X", "GET", String.format("http://localhost:9000/tables/%s_OFFLINE/size?detailed=false", tableName)
+        "curl",
```
Comment (cyrilou242) [inline pinot-test-container/src/main/java/org/apache/pinot/testcontainer/PinotContainer.java:175]: it seems we don't have the same code style configuration.
let's loop with Suvodeep and Shounak to ensure everybody has the same.

Comment (cyrilou242) [reply pinot-test-container/src/main/java/org/apache/pinot/testcontainer/PinotContainer.java:175]: not a blocker of course

**Review by cyrilou242 — APPROVED** [review]

**Comment by cyrilou242** [comment]
lgtm

File: pinot-test-container/src/main/java/org/apache/pinot/testcontainer/PinotContainer.java (hunk 1 above)
Comment (jacksonargo) [reply pinot-test-container/src/main/java/org/apache/pinot/testcontainer/PinotContainer.java:175]: 🤷 I imported the styles from https://github.com/startreedata/thirdeye/blob/master/thirdeye-checkstyle.xml, but ya Intellij still didn't like this line.

**Pull request merged**